app/models/manager_refresh/inventory_object.rb

```diff
@@ -81,45 +81,22 @@ def dependency?
       true
     end
 
-    def allowed_writers
-      return [] unless model_class
-
```

Comment:
I would deprecate the automatic way, once there is some gain from adding these attributes manually